CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to the vCon project! We value your time and effort and are excited to have you join our community. Please read through this document to understand the guidelines and processes for contributing to this project.
+
+## Table of Contents
+
+1. [Code of Conduct](#code-of-conduct)
+2. [Getting Started](#getting-started)
+3. [How to Contribute](#how-to-contribute)
+   - [Reporting Bugs](#reporting-bugs)
+   - [Suggesting Enhancements](#suggesting-enhancements)
+   - [Pull Requests](#pull-requests)
+4. [Style Guidelines](#style-guidelines)
+   - [Coding Style](#coding-style)
+   - [Commit Messages](#commit-messages)
+5. [Additional Resources](#additional-resources)
+
+## Code of Conduct
+
+This project and everyone participating in it are governed by our [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [ghostofbasho@gmail.com].
+
+## Getting Started
+
+To get started with contributing to this project, follow these steps:
+
+1. Fork the repository on GitHub.
+2. Clone your fork locally by running `git clone https://github.com/YOUR-USERNAME/vcon.git`.
+3. Add the original repository as a remote by running `git remote add upstream https://github.com/vcon-dev/vcon.git`.
+4. If you've added a new feature or fixed a bug, make sure to add appropriate tests.
+5. Ensure that the entire test suite passes by running the tests as specified in the project documentation.
+6. Update the documentation as necessary to reflect your changes.
+
+## How to Contribute
+
+### Reporting Bugs
+
+If you find a bug in the project, please open a new issue on the project's GitHub repository. When reporting a bug, please include:
+
+- A clear and descriptive title.
+- A detailed description of the bug, including steps to reproduce it.
+- The expected behavior and the actual behavior you encountered.
+- Any relevant code snippets, error messages, or screenshots.
+
+### Suggesting Enhancements
+
+If you have an idea for a new feature or improvement, please open a new issue on the project's GitHub repository. When suggesting enhancements, please include:
+
+- A clear and descriptive title.
+- A detailed description of the proposed enhancement, including examples or use cases.
+- Any relevant technical details or proposed implementation, if applicable.
+
+### Pull Requests
+
+To submit a pull request with your changes, follow these steps:
+
+1. Ensure your changes adhere to the project's style guidelines.
+2. Fetch the latest changes from the upstream repository by running `git fetch upstream`.
+3. Rebase your branch on top of the latest upstream changes by running `git rebase upstream/main`.
+4. Push your changes to your fork by running `git push origin BRANCH_NAME`.
```

Comment:
Do they need a "git branch BRANCH_NAME" and "git checkout BRANCH_NAME" and "git commit ..." steps?